include/deal.II/base/std_cxx11/type_traits.h

```diff
@@ -77,6 +79,9 @@ namespace std_cxx11
                               boost::has_trivial_constructor<T>::value &&
                               boost::has_trivial_destructor<T>::value;
   };
+
+  using boost::true_type;
+  using boost::false_type;
```

Comment:
Did you test this with a non-C++11 compiler? Don't we need an `#include` to import these classes?